chroma_core/models/stratagem.py

```diff
@@ -317,153 +318,33 @@ def get_deps(self):
 
 class RunStratagemStep(Step):
     def run(self, args):
-        host = args["host"]
-        path = args["path"]
-        target_name = args["target_name"]
-        report_duration = args["report_duration"]
-        purge_duration = args["purge_duration"]
-        search_expression = args["search_expression"]
-        action = json.loads(args["action"])
-
-        def calc_warn_duration(report_duration, purge_duration):
-            if report_duration is not None and purge_duration is not None:
-                return "&& != type S_IFDIR && < atime - sys_time {} > atime - sys_time {}".format(
-                    report_duration, purge_duration
-                )
+        pass
 
-            return "&& != type S_IFDIR < atime - sys_time {}".format(report_duration or 0)
-
-        def get_body(mount_point, action, report_duration, purge_duration, search_expression):
```

Comment:
I may be missing it, but I'm not seeing this getting generated anywhere by this patch.  Specifically LAT_COUNTER_INC (etc.) are not being generated by the serialization code in iml-wire-types (where I assume the conf file is getting generated)

Reply:
That is being done here (the file is large so github hid it's contents by default): 

https://github.com/whamcloud/integrated-manager-for-lustre/pull/2328/files/81ded592c199c59bd0c92f0a974bd65e47cbc2dd#diff-f138cff3f473efc07fd27907a450b2c0e5583cd133dcecec863e2a7d1d881d51R350-R378